src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -293,13 +294,28 @@ def complete(self, jobs):
                                 logOutput += "\n\n%s\n" % exMsg
 
                             if condorFile == condorLog:
-                                for condorReason in BasicAlgos.findMagicStr(condorFilePath, '<a n="Reason">'):
-                                    logOutput += condorReason
-                                    if "SYSTEM_PERIODIC_REMOVE" in condorReason or "via condor_rm" in condorReason:
-                                        exitCode = 99400
-                                        exitType = "RemovedByGLIDEIN"
+                                # for condor log, search for the information
+                                for matchObj in getIterMatchObjectOnRegexp(condorFilePath, CONDOR_LOG_FILTER_REGEXP):
+                                    condorReason = matchObj.group("Reason")
+                                    if condorReason:
+                                        logOutput += condorReason
+                                        if "SYSTEM_PERIODIC_REMOVE" in condorReason or "via condor_rm" in condorReason:
+                                            exitCode = 99400
+                                            exitType = "RemovedByGLIDEIN"
+                                        else:
+                                            exitCode = 99401
+
+                                    siteName = matchObj.group("Site")
```

Comment:
I think we should search for the complete class ad `MachineAttrGLIDEIN_CMSSite0`, just to avoid issues.

Reply:
It does. This is just key I added for matching the regular expression.

https://github.com/dmwm/WMCore/pull/8247/files/5bd0793abb2c05de40ab0b9d6e39edcfa1c56ae0#diff-0ed75c44d8f6712727213d921d2f53e4R53